pkgs/applications/networking/p2p/freenet/default.nix

```diff
@@ -1,15 +1,7 @@
-{ lib, stdenv, fetchurl, jdk, bash, coreutils, substituteAll, nixosTests }:
+{ lib, stdenv, fetchurl, jdk, bash, coreutils, substituteAll, nixosTests, jna }:
 
 let
   version = "build01494";
-  jna = fetchurl {
```

Comment:
I believe this deserves a better description. Technically the expression still uses JNA, but instead of a bundled/precompiled one we are compiling it locally.

Reply:
Do you mean the commit message itself?

Reply:
Yes, the commit message should be more elaborated.

Something like "freenet: use JNA provided by Nixpkgs instead of the upstream bundled one". Sorry, my English skills are subpar today...

Reply:
Done